crates/biome_css_analyze/tests/specs/nursery/noShorthandPropertyOverrides/invalid.css

```diff
@@ -0,0 +1,43 @@
+a { padding-left: 10px; padding: 20px; }
+
+a { border-width: 20px; border: 1px solid black; }
+
+a { border-color: red; border: 1px solid black; }
+
+a { border-style: dotted; border: 1px solid black; }
+
+a { border-image: url("foo.png"); border: 1px solid black; }
+
+a { border-image-source: url("foo.png"); border: 1px solid black; }
+
+a { pAdDiNg-lEfT: 10Px; pAdDiNg: 20Px; }
+
+a { PADDING-LEFT: 10PX; PADDING: 20PX; }
+
+/* Overriding within nested rule */
+a { padding-left: 10px; { b { padding-top: 10px; padding: 20px; }}}
+
+a { border-top-width: 1px; top: 0; bottom: 3px; border: 2px solid blue; }
+
+a { transition-property: opacity; transition: opacity 1s linear; }
+
+a { background-repeat: no-repeat; background: url(lion.png); }
+
+/* @media (color) { background-repeat: no-repeat; background: url(lion.png); } */
```

Comment:
For media queries, the property seems to be `CssCompoundSelector` instead of `CssGenericProperty`. Can we support media queries?

```
CssMediaAtRule {
    media_token: MEDIA_KW@793..799 "media" [] [Whitespace(" ")],
    queries: CssMediaQueryList [
        CssMediaConditionQuery {
            condition: CssMediaFeatureInParens {
                l_paren_token: L_PAREN@799..800 "(" [] [],
                feature: CssQueryFeatureBoolean {
                    name: CssIdentifier {
                        value_token: IDENT@800..805 "color" [] [],
                    },
                },
                r_paren_token: R_PAREN@805..807 ")" [] [Whitespace(" ")],
            },
        },
    ],
    block: CssRuleBlock {
        l_curly_token: L_CURLY@807..809 "{" [] [Whitespace(" ")],
        rules: CssRuleList [
            CssQualifiedRule {
                prelude: CssSelectorList [
                    CssCompoundSelector {
                        nesting_selector_token: missing (optional),
                        simple_selector: CssTypeSelector {
                            namespace: missing (optional),
                            ident: CssIdentifier {
                                value_token: IDENT@809..826 "background-repeat" [] [],
                            },
                        },
                        sub_selectors: CssSubSelectorList [
                            CssPseudoClassSelector {
                                colon_token: COLON@826..828 ":" [] [Whitespace(" ")],
                                class: CssPseudoClassIdentifier {
                                    name: CssIdentifier {
                                        value_token: IDENT@828..837 "no-repeat" [] [],
                                    },
                                },
                            },
                        ],
                    },
                    missing separator,
                    CssBogusSelector {
                        items: [
                            SEMICOLON@837..839 ";" [] [Whitespace(" ")],
                        ],
                    },
                    missing separator,
                    CssCompoundSelector {
                        nesting_selector_token: missing (optional),
                        simple_selector: CssTypeSelector {
                            namespace: missing (optional),
                            ident: CssIdentifier {
                                value_token: IDENT@839..849 "background" [] [],
                            },
                        },
                        sub_selectors: CssSubSelectorList [
                            CssPseudoClassSelector {
                                colon_token: COLON@849..851 ":" [] [Whitespace(" ")],
                                class: CssPseudoClassIdentifier {
                                    name: CssIdentifier {
                                        value_token: IDENT@851..854 "url" [] [],
                                    },
                                },
                            },
                        ],
                    },
                    missing separator,
                    CssBogusSelector {
                        items: [
                            L_PAREN@854..855 "(" [] [],
                        ],
                    },
                    missing separator,
                    CssCompoundSelector {
                        nesting_selector_token: missing (optional),
                        simple_selector: CssTypeSelector {
                            namespace: missing (optional),
                            ident: CssIdentifier {
                                value_token: IDENT@855..859 "lion" [] [],
                            },
                        },
                        sub_selectors: CssSubSelectorList [
                            CssClassSelector {
                                dot_token: DOT@859..860 "." [] [],
                                name: CssCustomIdentifier {
                                    value_token: IDENT@860..863 "png" [] [],
                                },
                            },
                        ],
                    },
                    missing separator,
                    CssBogusSelector {
                        items: [
                            R_PAREN@863..864 ")" [] [],
                            SEMICOLON@864..866 ";" [] [Whitespace(" ")],
                        ],
                    },
                ],
                block: CssBogusBlock {
                    items: [],
                },
            },
        ],
        r_curly_token: R_CURLY@866..867 "}" [] [],
    },
}
```

Reply:
Sorry. The syntax of the media query to test was wrong. After fixing it, I was able to handle it with `CssGenericProperty`. Just wanted to thank you for your new expertise!